platform/mbed_error_report.h

```diff
@@ -0,0 +1,50 @@
+
+/** \addtogroup platform */
+/** @{*/
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_ERROR_REPORT_H
+#define MBED_ERROR_REPORT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* routine to report the error */
```

Comment:
It's not doxy.

Reply:
Yes, :-). Its not really intended to be called directly. Its more for platform components to use so keeping non-doxy until we really find a need to expose it.